invenio_app_ils/ui/.env

```diff
@@ -0,0 +1,6 @@
+REACT_APP_JWT_TOKEN='token'
+REACT_APP_USER_ID=1
+REACT_APP_LOCATION_ID=1
+REACT_APP_USER_ROLE=librarian
+REACT_APP_JWT_TOKEN_EXPIRATION=11111111111111111111111
+REACT_APP_JWT_USERNAME=librarian
```

Comment:
I am not sure we have an username...

Reply:
I think we had added it in some point but the backend code is not there anymore...I just left whatever was there just not to break UI for now...This file will go away once we implement the oauth workflow during the sprint.